contracts/mainnet/connectors/morpho-aave-v3/helpers.sol

```diff
@@ -0,0 +1,36 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+import "./interface.sol";
+import "../../common/stores.sol";
+import "../../common/basic.sol";
+import "../../common/interfaces.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+abstract contract Helpers is Stores, Basic, Ownable {
```

Comment:
We can remove Ownable

Reply:
Reason - https://github.com/Instadapp/dsa-connectors/pull/283#discussion_r1120926565